tests/serialise/xml-serialise/ietf-json-serialise.yang

```diff
@@ -0,0 +1,256 @@
+module ietf-json-serialise {
```

Comment:
We should probably name this module per the convention in the other tests -- i.e.,`xml-serialise`.